sig/graft/hook_point.rbs

```diff
@@ -66,7 +66,7 @@ module Graft
 
     def disabled?: (String key) -> bool
 
-    module Prepend
+    module Prepend : HookPoint
```

Comment:
What does this mean? 

Does it mean `HookPoint` class will include this module?

Reply:
This _self type_ parameter constraints the `Prepend` module to only be mixed in `HookPoint`, therefore `Prepend` can assume having access to `HookPoint` methods.